### PR TITLE
4530 Create UI Home

### PR DESCRIPTION
**Pull request by TungPhanThanh**
create base class, UI, widget main screen
https://edu-redmine.framgia.vn/issues/4530

### CODE REVIEW

**Comment by TungPhanThanh** [comment]
ready

File: app/src/main/java/com/example/themo/musicmarvelous/data/repository/TrackRepository.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        return sInstance;
+    }
+
+    @Override
+    public void getOfflineTracks(TrackDataSource.OnFetchDataListener<Track> listener) {
+
+    }
+
+    @Override
+    public void getOnlineTracks(String genre, int limit, int offSet, TrackDataSource.OnFetchDataListener<Track> listener) {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/themo/musicmarvelous/data/repository/TrackRepository.java]: vượt 80 kí tự

File: app/src/main/java/com/example/themo/musicmarvelous/ui/main/MainActivity.java [hunk 2]
```diff
@@ -1,15 +1,48 @@
...
+        @Override
+        public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
+            Fragment fragment;
+            switch (menuItem.getItemId()) {
+                case R.id.navigation_home:
+                    return true;
+                case R.id.navigation_music:
+                    return true;
+                case R.id.navigation_settings:
+                    return true;
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/themo/musicmarvelous/ui/main/MainActivity.java:43]: add default case.

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,33 +1,53 @@
...
 
-        <android.support.v7.widget.Toolbar
-            android:layout_width="match_parent"
+        <ImageView
+            android:id="@+id/image_dehaze"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:background="@color/color_yellow_dark">
+            android:layout_marginRight="@dimen/dp_10"
+            android:paddingTop="5dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_main.xml]: hardcode

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -1,33 +1,53 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_main.xml]: Sao  lại không dùng ConstrainLayout

File: app/src/main/res/layout/fragment_home.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <android.support.v7.widget.RecyclerView
+        android:id="@+id/recyclerview_genre"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_home.xml]:
```suggestion
        android:id="@+id/recycler_genre"
```

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by TungPhanThanh** [comment]
ready

**Pull request merged**